core/src/toga/window.py

```diff
@@ -996,12 +988,29 @@ def __init__(
             closeable=closeable,
         )
 
+        # Create a toolbar that is linked to the app. Only install a change listener if
+        # the main app has a listener - this is an indicator that the app has finished
+        # starting up.
+        self._toolbar = CommandSet(
+            on_change=(
+                self._impl.create_toolbar if self.app.commands.on_change else None
+            ),
+            app=self.app,
+        )
+
     @property
     def _default_title(self) -> str:
         return toga.App.app.formal_name
```

Comment:
Agreed on simplifying the title.

Not sure there's much we can do about the constructor though. We still need the constructor to do the toolbar setup; but if we reduce the prototype to `(*args, **kwargs)`, that's what is reflected in the documentation as the prototype of the constructor. If the cost of clear documentation is a little bit of code duplication, I'm happy to live with that.

Reply:
Isn't it clear enough to say "accepts the same arguments as `Window`"? We don't repeat all the base class properties and methods, so why should we repeat the constructor arguments?

This would also save us from having to manually edit both classes to sync any changes to the arguments, defaults or docstring. In fact, they have already gone out of sync, because the Window docstring hasn't been updated for the new default title.

The change to the default title should also have a change note.

Reply:
Except they're not *exactly* the same - MainWindow doesn't accept `closable`, and for historical reasons, the order is *very* slightly different. I agree there's effort required to keep them in sync; but that's true of any documentation really.

I've modified the existing Window docstring; and I've added a new change note.

Reply:
As discussed: closable *should* exist (at least, it needs to in preparation for supporting simple windows). On that basis, I've simplified the prototype to be as bare as it can be; this is how the entire MainWindow reference docs render:
<img width="604" alt="Screenshot 2024-06-20 at 7 02 10 AM" src="https://github.com/beeware/toga/assets/37345/653aefcf-e220-417a-845c-3450907a0e54">